src/pages/Account.tsx

```diff
@@ -10,14 +10,14 @@ import Button from '../components/button';
 import Chip from '../components/chip';
 import ErrorText from '../components/form/ErrorText';
 import Input from '../components/form/Input';
-import Select from '../components/form/Select';
 import Label from '../components/typography/Label';
 import Title from '../components/typography/Title';
 import useUser from '../hooks/useUser';
 import { FlexColumn, FlexRow } from '../layout/Layout.styled';
 import { useAppStore } from '../store';
 import { AuthUser } from '../types/AuthUserType';
 import { GetGroupsResponse } from '../types/GroupType';
+import Select from '../components/select';
 
 const ACADEMIC_CREDENTIALS = ['Bachelors', 'Masters', 'PhD', 'JD', 'None', 'Other'];
```

Comment:
Remove `Other` from the array as we can add custom options nos, not sure about the None